tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -788,11 +788,13 @@ def index_mass_quantile(x, c, param):
     s = sum(np.abs(x))
     res = pd.Series()
 
+    x = pd.Series(x)
+
     if s == 0: # all values in x are zero or it has length 0
         for config in param:
             res["{}__index_mass_quantile__q_{}".format(c, config["q"])] = np.NaN
     else: # at least one value is not zero
-        mass_centralized = np.cumsum(np.abs(x)) * 1.0 / s
+        mass_centralized = (np.cumsum(np.abs(x)) * 1.0 / s).values
```

Comment:
I would add the `.value`to the x inside the abs call. I think it is more readable